examples/cpp/CMakeLists.txt

```diff
@@ -4,7 +4,28 @@ set(SOURCE "main.cpp"
            "how_to_part_02_detections.cpp"
 )
 
+set(kwiver_examples_libraries vital_algo
+                              kwiver_algo_ocv
+                              kwiver_algo_vxl)
+
+if(fletch_ENABLED_YAMLCPP)
+  # YAML parsing and raw kpf examples
+  find_package( yaml-cpp REQUIRED )
+  list(APPEND SOURCE kpf/kpf_example.cxx
+                     kpf/kpf_example_simple.cxx
+                     kpf/kpf_example_complex.cxx
+                     kpf/kpf_text_reader.cxx
+                     kpf/kpf_yaml_reader.cxx
+                     kpf/yaml_parser.cxx
+                     kpf/kpf_example_yaml.cxx
+  )
+
+  list(APPEND kwiver_examples_libraries kwiver_algo_kpf kpf_yaml ${YAML_CPP_LIBRARIES})
+  if(WIN32)
+    set(kwiver_examples_flags YAML_CPP_DLL)
+  endif()
+endif()
+
```

Comment:
The target(s) in here which include <yaml-cpp/yaml.h> need access to the include_dirs for yaml. You will need to copy the target_include_dirs call from the arrows/kpf CMakeLists.txt here, or change that one so it's just include_directories and move it somewhere that allows scoping to carry the include_directories everywhere it's needed.